e2e/tests/plugins/notebook/addRestrictedNotebook.js

```diff
@@ -0,0 +1,6 @@
+(function () {
```

Comment:
Can you add some documentation on how this is called?

Reply:
I can. Will I? Yes. I will.